README.rst

```diff
@@ -40,6 +40,83 @@ Single-node influxdb, SSL for http frontend:
               key_file: /etc/influxdb/ssl/key.pem
               cert_file: /etc/influxdb/ssl/cert.pem
 
+Single-node influxdb with an admin user:
+
+.. code-block:: yaml
+
+    influxdb:
+      server:
+        enabled: true
+        http:
+          enabled: true
+          bind:
+            address: 0.0.0.0
+            port: 8086
+        admin:
+          enabled: true
+          bind:
+            address: 0.0.0.0
+            port: 8083
+          user:
+            enabled: true
+            name: root
+            password: secret
+
+Single-node influxdb with new users:
+
+.. code-block:: yaml
+
+    influxdb:
+      server:
+        users:
+          user1:
+            enabled: true
+            admin: true
+            name: username1
+            password: keepsecret1
+          user2:
+            enabled: true
+            admin: false
+            name: username2
+            password: keepsecret2
+
+Single-node influxdb with new databases:
+
+.. code-block:: yaml
+
+    influxdb:
+      server:
+        databases:
+          mydb1:
+            enabled: true
+            name: mydb1
+          mydb2:
+            enabled: true
+            name: mydb2
+
+Here is how to manage grants on database:
+
+.. code-block:: yaml
+
+    influxdb:
+      server:
+        grants:
+          username1_mydb1:
+            enabled: true
+            user: username1
+            database: mydb1
+            grant: all
```

Comment:
nit but the example could show different privileges (read, write and all for instance). Also instead of:

```
          username1_mydb1:
            enabled: true
            user: username1
            database: mydb1
            grant: all
```

it could be:

```
          username1_mydb1:
            enabled: true
            user: username1
            database: mydb1
            privilege: all
```

to map with the InfluxDB vocabulary.